content/chainguard/chainctl-usage/comparing-chainctl-to-console/index.md

```diff
@@ -0,0 +1,142 @@
+---
+title: "Compare chainctl usage with the Chainguard Console"
+linktitle: "chainctl vs The Console"
+type: "article"
+description: "A comparison of completing tasks using chainctl vs the Chainguard Console."
+date: 2025-06--2T11:07:52+02:00
+lastmod: 2025-06--2T11:07:52+02:00
+draft: false
+tags: ["Chainguard Control", "chainctl", "Chainguard Console", "Product", ]
+images: []
+menu:
+  docs:
+    parent: "chainctl-usage"
+weight: 025
+toc: true
+---
+
+When should I use the Chainguard Console? When is it better to use `chainctl`? This page gives some guidance on the benefits of each method for managing your Chainguard Containers to help you make that decision.
+
+
+## Prerequisites
+
+To access the [Chainguard Console](/chainguard/chainguard-images/how-to-use/images-directory/) you need to [create an account and sign in](https://console.chainguard.dev/auth/login). The Console is accessible to everyone, including users who aren't Chainguard customers.
+
+To use `chainctl`, start by [installing chainctl](/chainguard/chainctl-usage/how-to-install-chainctl/). See [Get Started with chainctl](/chainguard/chainctl-usage/getting-started-with-chainctl/) to help you begin using it; the examples on this page assume you have `chainctl` installed and are authenticated.
+
+
+## High-level Comparison
+
+The Console is especially useful for one-off information searches, such as when you don't know precisely what you want to know. It's also good at providing detailed information after a few clicks to zoom in. You can perform useful container-related query tasks from within the Console, as this page will demonstrate with some examples.
+
+If you know specifically what you are looking for or what you want to accomplish, `chainctl` is a powerful way to do so. It can perform some additional tasks that are not yet available in the Console, such as [comparing images with a diff](/chainguard/chainctl-usage/comparing-images/).
+
+
+## Example - Find Available Images
+
+To find the images available to you in the Console, do this:
+
+1. Open the [Console](https://console.chainguard.dev)
+![Screenshot showing the Overview page in the Console.](console-overview.png)
+
+1. On the Overview page that opens, click **Organization Images** in the sidebar.
+![Screenshot showing the Organization Images page in the Console, which lists all of the images available along with data for each including Status, Latest tag, Pull URL, and when the image was last updated.](console-org-images.png)
+
+
+To find the images available to you using `chainctl`, use this command. The list of available images is likely to be long and will scroll past you quickly in the terminal.
```

Comment:
Since the chainctl option is not very user-friendly, especially if someone is an "all you can eat" images user, what about adding different flags or parameters to limit / sort and showing how people can build upon the command that way?